tests/test_commands.py

```diff
@@ -167,3 +164,59 @@ def to_item(self):
         json.loads(fixture.exception_path.read_bytes())["type_name"]
         == "web_poet.exceptions.core.UseFallback"
     )
+
+
+def test_savefixture_adapter(tmp_path) -> None:
+    project_name = "foo"
+    cwd = Path(tmp_path)
+    call_scrapy_command(str(cwd), "startproject", project_name)
+    cwd /= project_name
+    type_name = "foo.po.BTSBookPage"
+    (cwd / project_name / "po.py").write_text(
+        """
+from web_poet.pages import WebPage
+
+
+class BTSBookPage(WebPage):
+    async def to_item(self):
+        return {
+            'name': self.css("h1.name::text").get(),
+        }
+"""
+    )
+    (cwd / project_name / "settings.py").write_text(
+        """
+from collections import deque
+
+from itemadapter.adapter import DictAdapter, ItemAdapter
+
+
+class LowercaseDictAdapter(DictAdapter):
+    def __getitem__(self, field_name):
+        item = super().__getitem__(field_name)
+        if isinstance(item, str):
+            return item.lower()
+        return item
+
+
+class CustomItemAdapter(ItemAdapter):
+    ADAPTER_CLASSES = deque([LowercaseDictAdapter])
+
+
+SCRAPY_POET_TESTS_ADAPTER = CustomItemAdapter
+"""
+    )
+
+    with MockServer(ProductHtml) as server:
+        call_scrapy_command(
+            str(cwd),
+            "savefixture",
+            type_name,
+            f"{server.root_url}",
+        )
+    fixtures_dir = cwd / "fixtures"
+    fixture_dir = fixtures_dir / type_name / "test-1"
+    fixture = Fixture(fixture_dir)
+    assert fixture.is_valid()
+    item = json.loads(fixture.output_path.read_bytes())
+    assert item == {"name": "chocolate"}
```

Comment:
It seems we're not checking the item in other tests; would it make sense to check it, at least to ensure that the chocolate is not lower-case originally?

Reply:
Alternatively, you can just return a constant from the page object, without using mock server.